README.md

```diff
@@ -37,7 +37,7 @@ Out of the scope of this README, but in summary [something like this](https://ww
 
 ```bash
 docker build . -t robot-cow-bot
-docker run -d robot-cow-bot
+docker run -d --env-file .env robot-cow-bot
```

Comment:
in this pr u haven't added the instructions, or dotenv library. I like this change, but is this part of another PR?

Reply:
Well, now with the move to a different folder for some reason docker refuses to pick up the env file.
Had to explicitly set it as a `docker run` parameter to get it working, thus, added here.